crates/proof-of-sql/src/proof_primitive/dory/dory_commitment_helper_gpu.rs

```diff
@@ -94,7 +96,7 @@ fn compute_dory_commitment_impl<'a, T>(
 where
     &'a T: Into<DoryScalar>,
     &'a [T]: Into<Sequence<'a>>,
-    T: AsBytes + Copy + transpose::OffsetToBytes,
+    T: AsBytes + Copy + OffsetToBytes,
```

Comment:
Can you also change this to simply be:
```suggestion
    &'a [T]: Into<Sequence<'a>>,
    T: OffsetToBytes,
```

Reply:
Removing both `T: AsBytes + Copy` causes the compiler to complain. They are still needed because the trait `std::marker::Copy` and `AsBytes` is not implemented for `T` on the `column` parameter. It is required by a bound introduced by `transpose::transpose_for_fixed_msm` call. I will be removing this function in a follow up PR. Do you think it's worth figuring out a fix for this PR?

Reply:
Thanks for the help, it is removed!